.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Feautre] "
+labels: ''
+assignees: ''
+
+---
+
+**Feature**
+<!-- A clear and concise description of the feature proposal -->
+
+**Motivation**
+<!-- Please outline the motivation for the proposal. Is your feature request related to a problem? e.g., I'm always frustrated when [...]. If this is related to another GitHub issue, please link here too -->
+
+**Pitch**
+<!-- A clear and concise description of what you want to happen. -->
+
+**Alternatives**
+<!-- A clear and concise description of any alternative solutions or features you've considered if any. -->
+
+**Additional context**
+<!-- Add any other context or screenshots about the feature request here. -->
```

Comment:
With regard to the `Feature request`, I thought it may be useful for this project to also ask explicitly for existing online resources. papers, and/or people to contact. The reason is that often, specific methods may already have some implementation somewhere, and also sometimes there are scientists who are clear experts on a specific method.

Thus, I'd suggest to modify the last point to give contributors a bit more to think about:
```
<!-- Add any other context about the feature request here. This may be screenshots, online resources, potential experts to contact, scientific publications, demos, etc.-->
```